src/users/containers/user-details/user-details.component.ts

```diff
@@ -130,6 +130,10 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  public isSuspendedOrPending(status) {
+    return status === 'Suspended' || status === 'Pending';
```

Comment:
Maybe make this a bit more 'generic'? ```isInactive``` or something? Could also use an array value for the statuses instead

Reply:
@paddybasi  this property is set by PRD. Not in our hand

Reply:
No sorry, I meant the function naming and testing the status value:
```
public isInactive(status, inactiveStatuses: string[] = ['Suspended', 'Pending']) {
   return inactiveStatuses.includes(status);
}
```

Reply:
Done